var/spack/repos/builtin/packages/py-pyfftw/package.py

```diff
@@ -13,11 +13,12 @@ class PyPyfftw(PythonPackage):
     homepage = "http://hgomersall.github.com/pyFFTW"
     url      = "https://pypi.io/packages/source/p/pyFFTW/pyFFTW-0.10.4.tar.gz"
 
+    version('0.12.0', sha256='60988e823ca75808a26fd79d88dbae1de3699e72a293f812aa4534f8a0a58cb0')
     version('0.11.1', sha256='05ea28dede4c3aaaf5c66f56eb0f71849d0d50f5bc0f53ca0ffa69534af14926')
     version('0.10.4', sha256='739b436b7c0aeddf99a48749380260364d2dc027cf1d5f63dafb5f50068ede1a')
 
     depends_on('fftw')
-    depends_on('py-setuptools',    type='build')
-    depends_on('py-cython',        type='build')
-    depends_on('py-numpy@1.6:',    type=('build', 'run'))
-    depends_on('py-scipy@0.12.0:', type=('build', 'run'))
+    depends_on('py-setuptools',        type='build')
+    depends_on('py-cython@0.29:0.999', type='build')
+    depends_on('py-numpy@1.6:',        type=('build', 'run'), when='@:0.10.4')
+    depends_on('py-numpy@1.10:',       type=('build', 'run'), when='@0.11.0:')
```

Comment:
```suggestion
    depends_on('py-numpy@1.10:1.999',  type=('build', 'run'), when='@0.11.0:')
```
For the same reason mentioned above, `numpy>=1.10, <2.0` in the `setup.py`

Reply:
Is it 1.999 or 0.999? Sorry, I am still just learning the meaning of the code versions.

Reply:
Cython is 0.999, Numpy is 1.999.

Version ranges are _inclusive_ in Spack, so `1.10:1.999` is basically equivalent to `>= 1.10, < 2.0`

Reply:
Ah perfect! That makes total sense now.